packages/site/cms-site/src/blocks/PixelImageBlock.tsx

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-restricted-imports
```

Comment:
I thought about disabling the rule for `@comet/cms-site`, but decided against it, because there may be scenarios where we should use our wrappers in the library as well.